fractal_server/app/routes/api/v2/_aux_functions.py

```diff
@@ -107,9 +107,7 @@ async def _get_workflow_check_owner(
             detail=(f"Invalid {project_id=} for {workflow_id=}."),
         )
 
-    # Refresh so that workflow.project relationship is loaded (see discussion
-    # in issue #1063)
-    await db.refresh(workflow)
+    workflow = await db.get(WorkflowV2, workflow_id, populate_existing=True)
```

Comment:
1. Let's try to merge this statement with the similar one above
2. Let's review the SQL statements produced by this ORM call (with DB_ECHO=1). More specifically: is the `workflow.project.user_list` relationship loaded explicitly?